### PR TITLE
k8s

### PR DESCRIPTION
**Pull request by klml**
**Description**

Show all K8S apps on https://opensource.muenchen.de/de/software/openshift.html

### CODE REVIEW

File: de/software/openshift.md [hunk 1]
```diff
@@ -8,7 +8,20 @@ tags:
 - support
 - infrastruktur
 ---
-Auf der Kubernetes Plattform __Openshift__ werden ca. 100 interne, als auch externe Services betrieben.
+Auf der Kubernetes Plattform __Openshift__ werden interne, als auch externe Services betrieben.
```
Comment (eidottermihi) [inline de/software/openshift.md:11]: shouldn't we mention "Red Hat"? like 

Auf der Kubernetes Plattform "Red Hat OpenShift" ....

File: software/appswitcher.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+vi software/digiwf.md
+vi software/isi.md
+vi software/kafka.md
+vi software/kiwi.md
+vi software/kobit.md
+vi software/mdas.md
+vi software/openshift.md
+vi software/praktikumsplaner.md
+vi software/zammad.md
+vi software/zeitmanagementsystem.md--
```
Comment (eidottermihi) [inline software/appswitcher.md:12]: oopsie?

File: software/dave.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ tags:
 - application
 - eigenentwicklung
 - start2021
+  - k8s
```
Comment (eidottermihi) [inline software/dave.md:13]: indent

**Review by eidottermihi — CHANGES_REQUESTED** [review]

File: de/software/openshift.md (hunk 1 above)
Comment (klml) [reply de/software/openshift.md:11]: i would not want to name any brands

**Review by eidottermihi — APPROVED** [review]

**Pull request merged**